server/src/services/album.service.ts

```diff
@@ -174,7 +174,11 @@ export class AlbumService extends BaseService {
         albumThumbnailAssetId: album.albumThumbnailAssetId ?? firstNewAssetId,
       });
 
-      await this.eventRepository.emit('album.update', { id, updatedBy: auth.user.id });
+      const allUsersExceptUs = [...album.albumUsers.map((au) => au.user.id), album.owner.id].filter(
+        (userId) => userId !== auth.user.id,
+      );
+
+      await this.eventRepository.emit('album.update', { id, recipientIds: allUsersExceptUs });
```

Comment:
You can check if `allUsersExceptUs` is empty before emitting in the first place.